src/sentry/incidents/charts.py

```diff
@@ -189,7 +189,7 @@ def build_metric_alert_chart(
     aggregate = translate_aggregate_field(snuba_query.aggregate, reverse=True)
     # If we allow alerts to be across multiple orgs this will break
     project_id = snuba_query.subscriptions.first().project_id
-    time_window_minutes = snuba_query.time_window / 60
+    time_window_minutes = snuba_query.time_window // 60
```

Comment:
Do we always want it floored? assuming yes

Reply:
yeah always